x-pack/filebeat/module/envoyproxy/README.md

```diff
@@ -0,0 +1,128 @@
+# Envoyproxy Module
+
+This is a filebeat module for envoy proxy. 
+
+## Caveats
+
+* Module is to be considered _beta_.
+
+## Download and install Filebeat
+
+Grab the filebeat binary from elastic.co, and install it by following the instructions.
+
+## Deployment Scenario #1: envoy native deployment
+
+Make sure to update envoy configuration to enable log plugin. This module assumes that envoy log
```

Comment:
sorry, probably my lack of knowledge of Envoy, but isn't it writing to stderr by default? there is a flag that sends logs to a file. Other than that I'm not aware of logs plugins. If I were a user a link to the envoy docs about configuring the plugin would be much appreciated.

It might happen that I don't have the background to know about those log plugins and it might be too obvious for an Envoy user. In that case, please, dismiss my comment :-)

Reply:
This was based on the coredns module that I did earlier. I forgot to remove this part. :-)